src/v2/Apps/Fairs/Components/FairsFairBanner.tsx

```diff
@@ -0,0 +1,132 @@
+import { Box, BoxProps, Image, ResponsiveBox, Text } from "@artsy/palette"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+import styled from "styled-components"
+import { RouterLink } from "v2/Artsy/Router/RouterLink"
+import { FairsFairBanner_fair } from "v2/__generated__/FairsFairBanner_fair.graphql"
+
+const Overlay = styled(Box)`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  display: flex;
+  align-items: flex-end;
+  background-color: rgba(0, 0, 0, 0.5);
+  opacity: 0;
+  transition: opacity 250ms;
+
+  &:hover {
+    opacity: 1;
+  }
+`
+
+interface FairsFairBannerProps extends BoxProps {
+  fair: FairsFairBanner_fair
+}
+
+const FairsFairBanner: React.FC<FairsFairBannerProps> = ({ fair, ...rest }) => {
+  const banner =
+    fair.image &&
+    (fair.bannerSize === "x-large" ? fair.image.large : fair.image.small)
+
+  const icon = fair?.profile?.icon?.resized
+
+  return (
+    <Box {...rest}>
+      <RouterLink
+        to={fair.href}
+        style={{ display: "block", textDecoration: "none" }}
```

Comment:
(general) we add `textDecoration: none` everywhere this is used -- we should probably reverse this and opt out by default and opt if needed.

Reply:
I concur — there's a `noUnderline` prop but it doesn't work correctly if you want to also add display options. We should revisit this component a bit and mixin some styled-system functions